platform/linux-dpdk/pool/dpdk.c

```diff
@@ -194,7 +194,7 @@ static void
 odp_dpdk_mbuf_ctor(struct rte_mempool *mp,
 		   void *opaque_arg,
 		   void *raw_mbuf,
-		   unsigned i)
+		   unsigned i ODP_UNUSED)
```

Comment:
Since this is an internal routine, why not just drop the parameter rather than marking it `ODP_UNUSED` if it's no longer needed?

Reply:
This is a combination of rte_pktmbuf_init in rte_mbuf.c . So the style needs to the same as rte_pktmbuf_init in dpdk , otherwise there would be some error.

Reply:
Ok.